connectors/services/base.py

```diff
@@ -111,6 +113,27 @@ def raise_if_spurious(self, exception):
         self.errors[0] = errors
         self.errors[1] = first
 
+    def _parse_connectors(self):
+        connectors = {}
+        configured_connectors = deepcopy(self.config.get("connectors"))
```

Comment:
The same copy of `config` is parsed to `JobExecutionService` and `JobSchedulingService` and `JobCleanupService`, so it has to be copied instead of processing the original copy.

`self.config.get("connectors").copy()` will just do a shallow copy, for this case, `connectors` is an array, and the items inside will be copied as the reference.